protocol-dashboard/index.html

```diff
@@ -3,10 +3,10 @@
 
 <head>
   <meta charset="utf-8" />
-  <link rel="shortcut icon" href="./favicon.ico" />
+  <link rel="shortcut icon" href="%VITE_DASHBOARD_BASE_URL%favicon.ico" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <meta name="theme-color" content="#CC0FE0" />
-  <link rel="manifest" href="./manifest.json" />
+  <link rel="manifest" href="%VITE_DASHBOARD_BASE_URL%manifest.json" />
```

Comment:
does this work if VITE_DASHBOARD_BASE_URL is "/", which is the default case? thought prod was broken unless we used relative paths with an explicit "./" base

Reply:
i think it works with `/` because `npm run build:prod && npm run serve` works. i also thought it was broken without relative paths so i'll test on IPFS before merging

Reply:
sounds good. for what its worth iirc `npm run build:prod && npm run serve` also worked for me with the `href="./..."` stuff but didnt work after actually deploying to IPFS